src/ui/InputText/InputText.component.tsx

```diff
@@ -8,6 +10,7 @@ function InputText({ onChange}: InputTextProps) {
       type="text" 
       placeholder="Ingresar tarea..."
       onChange={(event) => onChange(event.target.value)}
+      className={`${style.container}`}
```

Comment:
no need for backticks you could pass the variable directly as it's a string already

```diff
-      className={`${style.container}`}
+      className={style.container}
```